vint/encodings/decoder.py

```diff
@@ -2,12 +2,12 @@
 from typing import Dict, Any
 from pprint import pformat
 from pathlib import Path
-from vint.encodings.decoding_strategy.abstract import DecodingStrategy
-from vint.encodings.decoding_strategy.scriptencoding import DecodingStrategyByScriptencoding
-from vint.encodings.decoding_strategy.utf8 import DecodingStrategyForUTF8
-from vint.encodings.decoding_strategy.composed import ComposedDecodingStrategy
-from vint.encodings.decoding_strategy.empty import DecodingStrategyForEmpty
-from vint.encodings.decoding_strategy.chardet import DecodingStrategyByChardet
+from vint.encodings.decoding_strategy.abstract_strategy import DecodingStrategy
+from vint.encodings.decoding_strategy.scriptencoding_strategy import DecodingStrategyByScriptencoding
+from vint.encodings.decoding_strategy.utf8_strategy import DecodingStrategyForUTF8
+from vint.encodings.decoding_strategy.composed_strategy import ComposedDecodingStrategy
+from vint.encodings.decoding_strategy.empty_strategy import DecodingStrategyForEmpty
+from vint.encodings.decoding_strategy.chardet_strategy import DecodingStrategyByChardet
```

Comment:
What about having them all in `vint.encodings.decoding_strategy` directly?

Reply:
I improved the code on ee265da